src/libraries/Microsoft.Extensions.Options/ref/Microsoft.Extensions.Options.cs

```diff
@@ -121,7 +121,7 @@ public partial interface IOptionsMonitor<[System.Diagnostics.CodeAnalysis.Dynami
     {
         TOptions CurrentValue { get; }
         TOptions Get(string? name);
-        System.IDisposable OnChange(System.Action<TOptions, string?> listener);
+        System.IDisposable? OnChange(System.Action<TOptions, string?> listener);
```

Comment:
Don't you also need to change

https://github.com/dotnet/runtime/blob/6d8cc6f1dd0152cf8454889daebdd3fcccdf3792/src/libraries/Microsoft.Extensions.Options/src/OptionsMonitorExtensions.cs#L20-L23

Reply:
Missed it, thanks